example/install_src.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+PWD=$(pwd)
+MYDIR=${1:-$PWD}
+if [ ! -d $MYDIR ]; then
+    echo "[Error] working directory=$MYDIR doesn't exist"
+    exit -1
+fi
+
+pushd $MYDIR
+
+WORKDIR=$(pwd)  # Absolute Path
+
+
+set -e
+set -x
+
+BRANCH=${2:-master} # can be a tag like 1.0.0
+
+
+
+RACKHD_CORE_REPOS=("on-core" "on-tasks" "di.js" )
+RACKHD_DEP_REPOS=("on-http" "on-taskgraph" "on-dhcp-proxy" "on-tftp" "on-syslog" )
```

Comment:
sorry about previous comments cause confusing, I mean on-core,on-tasks,di.js are depenency repos, and on-http/.. are main repos.